frontend/relay-workflows-lib/lib/components/SubmittedMessagesList.tsx

```diff
@@ -76,16 +93,53 @@ const renderSubmittedMessage = (
   }
 };
 
-interface SubmittedMessagesListProps {
-  submissionResults: (
+interface SubscribeAndRenderPropsList {
+  r:
```

Comment:
Maybe change these props from `r` and `i` to `result` and `index` to make it a bit easier to follow.